build_container/Dockerfile-ubuntu

```diff
@@ -10,7 +10,7 @@ SHELL ["/bin/ash", "-o", "pipefail", "-c"]
 RUN curl -fsSL https://github.com/multiarch/qemu-user-static/releases/download/v${QEMUVERSION}/qemu-${QEMUARCH}-static.tar.gz | tar zxvf - -C /usr/bin
 RUN chmod +x /usr/bin/qemu-*
 
-FROM ${IMAGEARCH}ubuntu:xenial
+FROM ${IMAGEARCH}ubuntu:bionic
```

Comment:
My impression is that the builds for ppc65le and aarch64 are still dependent on xenial/16.04

Reply:
Not really, those tools built for xenial can run on bionic, it doesn't block base image update.